packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -74,40 +74,36 @@ export class ChoiceGroupBase extends BaseComponent<IChoiceGroupProps, IChoiceGro
     const keyDefaultFocusable = keyChecked === undefined && firstEnabledOption ? firstEnabledOption.key : undefined;
 
     return (
-      // By default, we need to assign the role 'application' on the containing div
-      // because JAWS doesn't call OnKeyDown without this role
-      <div role={role} className={classNames.applicationRole}>
```

Comment:
This is a breaking change, we should retain this `<div>` in `6.x` _and_ allow for configurable `role` via prop.

The _only_ changes I would expect as part of this PR are: 

- The _removal_ of the JAWS comment on lines 77-78 assuming the JAWS bug it calls-out has since been fixed.
- The _removal_ of `'application'` as the default for `role` in the destructuring of props.

We _still_ want to support supplying a configurable `role` and `classNames.applicationRole` until _at least_ `7.0` as per our [Version Policy](https://github.com/OfficeDev/office-ui-fabric-react/#version-policy).

Reply:
Support was recently added in #6495.

Reply:
@KevinTCoughlin: Sure but I can't see any `role` being valid here, so we should remove it ASAP when we hit `7.0`.